robot/EDA/tests/browser/contacts/verify_preferred_phone.robot

```diff
@@ -0,0 +1,166 @@
+*** Settings ***
+Documentation
+...     Verify Preferred Phone functionality
+
+
+Resource        robot/EDA/resources/EDA.robot
+Library         DateTime
+Library         SeleniumLibrary
```

Comment:
I don't think you need to import SeleniumLibrary here. You're importing EDA.robot which imports Salesforce.robot, and that imports SeleniumLibrary.

Reply:
removed